app/lib/mappers/mapServices.js

```diff
@@ -22,7 +22,13 @@ function getDeliverer(deliverer) {
 }
 
 function mapServiceName(serviceTitle) {
-  return mapTitles[serviceTitle] || serviceTitle;
+  Object.keys(mapTitles).forEach((key) => {
+    if (new RegExp(key, 'i').test(serviceTitle) === true) {
```

Comment:
I'd suggest using the service's code, rather than using the name as the key to avoid the reg ex and do a match against a fixed string